pkg/acs/lib/calacs/acsdth.c

```diff
@@ -100,12 +100,9 @@ int AcsDth (char *in_list, char *output, int dthcorr, int printtime, int verbose
 	if (printtime)
     TimeStamp ("ACSDTH started", "");
 
-  sprintf(MsgText,"The task PyDrizzle needs to be run in order to generate");
-  trlmessage(MsgText);
-  sprintf(MsgText,"a geometrically corrected, drizzle-combined product.");
-  trlmessage(MsgText);
-  sprintf(MsgText,"PyDrizzle requires PyRAF. See pyraf.stsci.edu for more details.");
-  trlmessage(MsgText);
+  trlmessage("The task PyDrizzle needs to be run in order to generate\n"
+  	"a geometrically corrected, drizzle-combined product.\n"
+  	"PyDrizzle requires PyRAF. See pyraf.stsci.edu for more details.");
```

Comment:
Might as well update the message while we're at it?
```suggestion
  trlmessage("The task DrizzlePac needs to be run in order to generate\n"
  	"a geometrically corrected, drizzle-combined product.\n"
  	"See https://www.stsci.edu/scientific-community/software/drizzlepac for more details.");
```

Reply:
Yeah maybe :sweat_smile: